### PR TITLE
Priya

### LINKED ISSUE #92: Deletecomment can also be deleted by user who posted the post
Following Button margin ml-2 added
CreatPost Toast issue
Deletecomment can also be deleted by user who posted the post

### LINKED ISSUE #94: Delete Post icon disappearing when deleting comment
Delete Post icon disappearing when deleting comment.
Delete Comment button size and like heart style changes

### LINKED ISSUE #96: Comment is not showing user avatar in home and singleposts
Comment is not showing user avatar in home and singleposts 
changes done in posts.addComment api.

### PR DESCRIPTION
**Pull request by slpriya**
close #92 
close #94 
close #96 
Add Unfollow component on click of following button

### CODE REVIEW

**Review by Emily484 — APPROVED** [review]
looks good

File: client/src/components/Home/HomeSuggestion.js [hunk 1]
```diff
@@ -37,8 +37,9 @@ class HomeSuggestion extends Component {
                 <div className="make-gray left-15">Followed by {suggestion.followers.length} people</div>
               </div>
             </div>
+            {/* Follow component takes suggestion object as props */}
             <Follow
-              userId={suggestion._id}
+              user={suggestion}
```
Comment (solongoch) [inline client/src/components/Home/HomeSuggestion.js]: in Follow, it is taking userId from props, not user

File: client/src/components/Home/PostCommentItemHome.js [hunk 2]
```diff
@@ -31,15 +31,17 @@ render() {
...
-                    type="button"
-                    className="btn btn-danger btn-delete-comment"
-                    >
-                     <i className="fas fa-times fa-xs" />   
-                    </button>
-                ) : null}
+                      onClick={this.onDeleteClick.bind(this, postId, comment._id)}
+                      type="button"
+                      className="btn btn-danger btn-delete-comment fas fa-times shadow-none" 
+                      style={{padding : '3px 6px' , fontSize: '14px' }}
```
Comment (solongoch) [inline client/src/components/Home/PostCommentItemHome.js:40]: please avoid using inline styles

File: client/src/actions/profileActions.js [hunk 3]
```diff
@@ -109,16 +109,12 @@ export const deleteAccount = history => dispatch => {
...
-          type: SET_CURRENT_USER,
-          payload: {}
-        });
         dispatch(logoutUser());
       })
       .catch(err => {
         dispatch({
           type: GET_ERRORS,
-          payload: err.response.data
+          payload: {},
```
Comment (solongoch) [inline client/src/actions/profileActions.js:117]: why clearing errors here?

File: client/src/components/Home/HomeSuggestion.js (hunk 1 above)
Comment (slpriya) [reply client/src/components/Home/HomeSuggestion.js]: Yeah for get to change it

**Review by solongoch — APPROVED** [review]
Looks good

File: client/src/actions/profileActions.js (hunk 3 above)
Comment (slpriya) [reply client/src/actions/profileActions.js:117]: When u delete the account nothing should be there try delete the account in ur branch  we are getting error.

Comment (slpriya) [reply client/src/actions/profileActions.js:117]: When deleting account nothing should be there in redux. Try deleting in your local version of code getting error. and we are login out.

Comment (solongoch) [reply client/src/actions/profileActions.js:117]: yeah, but getting error means something happened to `delete account`. `err` should always be there.

**Review by slpriya — COMMENTED** [review]
Replied back

**Pull request merged**